### DISCUSSION TITLE
Using this module with Slack

### DISCUSSION THREAD

**Discussion by kawazoet** [OpenIDC/mod_auth_openidc#502] [General]
I don't know if it's possible to use _mod_auth_openidc_ to use Slack's application authentication infrastructure. Does anyone know?
To be specific, I don't know what to write in the parameter _OIDCProviderMetadataURL_.

**Comment by zandbelt** [comment]
I don't think Slack supports OpenID Connect

**Reply by kawazoet** [reply]
Thank you so much for your quick reply.
It turns out that I don't understand OpenID Connect and need to do learning.
